source/configuration_manual/authentication/static_password_database.rst

```diff
@@ -6,21 +6,23 @@ Static Password Database
 
 Static password database is typically used only for testing, proxying setups
 and perhaps some other special kind of setups. Static passdb allows all users
-to log in with any username. For password you return either:
+to log in with any username. For password you can either set `passdb_static_password` or set `nopassword = yes` in :dovecot_core:ref:`passdb_default_fields`:
```

Comment:
double-\`\`. Although passdb_static_password could perhaps be a ref to the setting, although right now it's in the next line ..but that's also a bit weird that this sentence ends with `:` as if it's going to show an example of how to use `fields { nopassword = yes }` but that's not there. Maybe move passdb_static_password elsewhere? Usually settings are listed in a separate section, although this time there would be just one setting.

Reply:
I moved the settings to core.rst now and added a :ref: here.